src/components/Error.astro

```diff
@@ -1,15 +1,16 @@
 ---
+/* eslint-disable tailwindcss/no-custom-classname */
```

Comment:
This is because the `expanded` class is used to open and close those attributes. Have left it for now but might cull it at a later date 👀